packages/react-hooks/src/useAnnouncer.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright IBM Corp. 2018, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { useEffect, useState } from 'react';
+import { usePortalNode } from './usePortalNode';
+
+/**
+ * Provides an `announce` method that allows a user to queue up an assertive or
+ * polite message to the user. This message is displayed in an `aria-live`
+ * region with the appropriate mode and the message is set as its text content.
+ * This `aria-live` region is the same for all components, so ordering of
+ * messages sent is important.
+ */
+export function useAnnouncer() {
+  const node = usePortalNode('carbon-announcer', node => {
```

Comment:
I'm curious if we want a callback to `usePortalNode`, or if we should just put this in an effect 🤔